Orangenspannung/2D-RPG
# P-03-Add-Sprites

# What has changed:
* Added Sprite resources
* Added some custom sprites for the fog since there was nothing comparable in the package
# How to test:
* Check out the resources, do they look fitting? is everything there for the project (Stairs will be used as Beginning/End)? 
* Check the fog and try not to die or something, check it as gif
